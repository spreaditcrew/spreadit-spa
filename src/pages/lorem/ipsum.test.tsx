import React from 'react';
import Ipsum from './ipsum';
import renderer from 'react-test-renderer';

describe('Routes', (): void => {
  test('something is happening', (): void => {
    const parameters: object = {
      router: { query: { id: 3 } },
    };

    const component = renderer.create(<Ipsum {...parameters} />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
