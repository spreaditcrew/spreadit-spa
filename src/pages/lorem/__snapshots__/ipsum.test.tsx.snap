// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes something is happening 1`] = `
<div
  className="block"
>
  Yes, it works! 
  3
</div>
`;
